modules/gitops-release-notes-1-2-1.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-2-1_{context}"]
+= Release notes for {gitops-title} 1.2.1
+
+{gitops-title} 1.2.1 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="support-matrix-1-2-1_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {gitops-title} 1.2.1
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
+| TP
+|===
+
+[id="fixed-issues-1-2-1_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, huge memory spikes were observed on the application controller on startup. The flag `--kubectl-parallelism-limit` for the application controller is now set to 10 by default, however
+this value can be overridden by specifying a number for `.spec.controller.kubeParallelismLimit` in the Argo CD CR spec.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1255]
+
+* The Kubernetes build failure caused by duplicate entries created in the kustomization.yaml when using the `kam bootstrap` command is now fixed. Pipeline has been updated to v0.24.2 and Tekton triggers have been updated to 0.14.2 to address this issue.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1273]
```

Comment:
GITOPS-1025 ? Issues does not seem to be correct, Am I missing something ?

Reply:
oh that is a mistake, I was using the link that was in the notes from last release so I got the format right and forgot to change both numbers, thanks for catching that. I'll fix it in all places